### PR TITLE
Feat: additional german translations

### PR DESCRIPTION
**Pull request by fstreicher**
- added new translations
- update some existing ones to better reflect functionality

Please see comments before merging, there's a couple of string I'm not quite sure about.

### CODE REVIEW

File: KlipTok.de-DE.json [hunk 1]
```diff
@@ -2,43 +2,115 @@
...
 	"About": "Über uns",
 	"Close": "Schließen",
 
 	// Streamer Page
-	"ClickToFollow": "Klicken um zu Folgen",
-	"ClickToUnfollow": "Klicken um zu Entfolgen",
-	"ActionOnTwitchTemplate": "{0} auf Twitch",
+	"ClickToFollow": "Auf Twitch Folgen",
+	"ClickToUnfollow": "Auf Twitch Entfolgen",
+	"ActionOnTwitchTemplate": "{0} auf Twitch", // {0} wird ersetzt mit "Follow" oder "Unfollow" -> "Follow on Twitch"
```
Comment (fstreicher) [inline KlipTok.de-DE.json:35]: Will the `(Un-)Follow` tokens be localized as well at some point?

File: KlipTok.de-DE.json [hunk 2]
```diff
@@ -2,43 +2,115 @@
...
+	"RetryButton": "Erneut versuchen",
+
+	// Follows page
+	"TwitchChannelsIFollow": "Kanäle, denen ich folge",
+	"NotLoggedIn": "Bitte melde dich an, um Kanäle anzuzeigen, denen du folgst.",
+
+	// Home Page
+	"DiscoverPopularClips": "Beliebte Clips entdecken",
+	"DiscoverClips": "Clips entdecken",
+	"InsertCoin": "Insert Coin",
```
Comment (fstreicher) [inline KlipTok.de-DE.json:64]: Not really sure what this si supposed to mean, appears to be some sort of a loading indicator?

File: KlipTok.de-DE.json [hunk 3]
```diff
@@ -2,43 +2,115 @@
...
+	"RightToForget": "Recht auf Vergessenwerden",
+	"DangerZone": "[ Gefahrenzone ]", // This is a label indicating the items beneath it make destructive changes to your record on the application 
+	"RightToForgetDescription": "Du hast das Recht, dass KlipTok alle Informationen, die Du geteilt hast, löscht. Mit einem Klick auf diesen Link bestätigst Du, dass KlipTok alle deine Kommentare, Likes und Clips entfernt.",
+	"RemoveMyContentFromKlipTok": "Meine Informationen und Inhalte von KlipTok löschen",
+	"ForgottenAsOfTemplate": "Daten gelöscht am: {0}", // {0} ist das Datum, an dem die Datenlöschung veranlasst wurde
+
+	// Single Clip Page
+	"MissingClipError": "KlipTok kennt diesen Clip nicht. Entweder existiert er nicht, oder der Twitch Kanal hat die Indizirung auf KlipTok abgewählt.",
+	"LikedBy": "Gefällt:",
+	"UserLikedTimeTemplate": "{0} gefällt dieser Clip", // {0} ist der Displayname des Nutzers, dem der Clip gefällt, {1} ist der relative Zeitstempel
```
Comment (fstreicher) [inline KlipTok.de-DE.json:94]: This is a bit of an ugly construct in german. For now I have opted to omit the timestampt and actually translate `like`, as I have done with the rest of the translations.
I could use an anglicism, which would make this sound a lot more natural with the timestamp, but I'm honestly not sure which would be preferred by users.

File: KlipTok.de-DE.json (hunk 1 above)
Comment (csharpfritz) [reply KlipTok.de-DE.json:35]: I'm not sure what your request is here...  The "Action on Twitch template" adds the action "Follow" or "Unfollow" where the token {0} is

File: KlipTok.de-DE.json (hunk 2 above)
Comment (csharpfritz) [reply KlipTok.de-DE.json:64]: Yes.. the "InsertCoin" is a loading indicator that references 1980's arcade machines

File: KlipTok.de-DE.json (hunk 1 above)
Comment (luke-777) [reply KlipTok.de-DE.json:35]: Why "Auf Twitch Folgen / Entfolgen" (Follow on Twitch)? Isn't this about Following on KlipTok?

Comment (fstreicher) [reply KlipTok.de-DE.json:35]: @luke-777 If you're referring to lines 33/34, those buttons actually modify your follows on Twitch, which KlipTok then mirrors.

@csharpfritz What I'm asking is basically: To me the comment reads, as if `{0}` is replaced by literals, and not localized strings, which would result in `Follow auf Twitch` (doesn't make much sense). So I assumed `Follow` and `Unfollow` will be tokenized to enable localization on those as well. I might be missing something obvious here, though.

File: KlipTok.de-DE.json [hunk 4]
```diff
@@ -2,43 +2,115 @@
...
-	"LastClipAdded": "Letzter Clip hinzugefügt:",
-	"NoProfilePicAvailable": "Kein Profilbild verfügbar"
+	"LastClipAdded": "Neuster Clip:",
+	"NoProfilePicAvailable": "Kein Profilbild verfügbar",
+
+
+	// Layout
+	"PublicBeta": "PUBLIC BETA",
+	"LoggedOutFromKlipTokAndTwitch": "Von KlipTok und Twitch ausgeloggt",
+	"LoginToKlipTokWithTwitch": "Mit Twitch bei KlipTon anmelden",
```
Comment (luke-777) [inline KlipTok.de-DE.json:49]: KlipTo**n**

File: KlipTok.de-DE.json [hunk 5]
```diff
@@ -2,43 +2,115 @@
...
+	"NoLikedClips": "Du hast noch bei keinem Clip 'Gefällt mir' geklickt!",
+	"TryLikeButtonTemplate": "Klicke unter einem Clip auf das Herz {0} and we'll report it here for you so you can easily find it again in the future",  // {0} ist das Herz Icon
+
+	// User Profile
+	"DoesNotKeepSeparateUserInformation": "KlipTok speichert keine Nutzerinformation abseits von Twitch, and wurde als Service entwickelt, um Twitch Clips zu erweitern.",
+	"AutoPlayVideosOnScroll": "Videos beim Scrollen automatisch abspielen",
+	"IndexClipsOfMyChannel": "Meine Clips indizieren",
+	"OptedOutDateTemplate": "Automatische Indizierung abgewählt am: {0}", // {0} ist das Datum des Opt-outs
+	"SaveMyChanges": "Änderungen speichern",
+	"SyncFollowersWithKlipTok":"Kanäle, denen ich folge mit, KlipTop synchronisieren",
```
Comment (luke-777) [inline KlipTok.de-DE.json:81]: Kommasetzung falsch "mit,"

File: KlipTok.de-DE.json (hunk 4 above)
Comment (fstreicher) [reply KlipTok.de-DE.json]: Thanks, fixed

File: KlipTok.de-DE.json (hunk 5 above)
Comment (fstreicher) [reply KlipTok.de-DE.json]: Thanks, fixed

File: KlipTok.de-DE.json (hunk 1 above)
Comment (csharpfritz) [reply KlipTok.de-DE.json:35]: No..  The {0} is replaced with the translation of 'ClickToFollow' and 'ClickToUnfollow' from lines 33 & 34

**Comment by fstreicher** [comment]
@csharpfritz ready to merge

**Comment by csharpfritz** [comment]
Thank you for the work on this.  I'll publish it shortly

**Pull request merged**